### ISSUE TITLE
Add some argument for train.py cli

### ISSUE THREAD

**Issue by tiena2cva** [GafBof/captcha_solver#3]
Some arg like:
- learning rate
- data folder
- batch size
- number of epoch
